shell/browser/ui/tray_icon_cocoa.mm

```diff
@@ -237,83 +103,44 @@ - (BOOL)getIgnoreDoubleClickEvents {
 }
 
 - (void)setTitle:(NSString*)title {
+  if ([title containsANSICodes]) {
+    [[statusItem_ button]
+        setAttributedTitle:[[title copy] attributedStringParsingANSICodes]];
```

Comment:
no need for the copy here

Reply:
Yeah, it returns a new string, in particular a `NSMutableAttributedString`. I think I put the `copy` there just in case.

Reply:
Right, i'll change that.